packages/api2/src/community/schemas/community.schema.ts

```diff
@@ -0,0 +1,78 @@
+import { ExposeId } from '@/shared/decorators/expose-id.decorator';
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { ApiProperty } from '@nestjs/swagger';
+import { Document, model, Types } from 'mongoose';
+import { mongoosePagination, Pagination } from 'mongoose-paginate-ts';
+import { IsBoolean, IsString, ValidateNested } from 'class-validator';
+import { CommunityBotStatus } from '../enums/community-bot-status';
+
+export type CommunityDocument = Community & Document;
+
+
+@Schema({ timestamps: true })
+export class Community {
+  constructor(partial?: Partial<Community>) {
+    if (partial) {
+      Object.assign(this, partial);
+    }
+  }
+
+  @ApiProperty({ example: '621f802b813dbdba9eeaf7b4', required: true })
+  @IsString()
+  @ExposeId()
+  _id: Types.ObjectId;
+
+  @ApiProperty({ example: 'banklessdao.givepraise.xyz', required: true })
+  @IsString()
+  @Prop({ type: String, required: true })
+  hostname: string;
+
+  @ApiProperty({ example: 'BanklessDAO', required: true })
+  @IsString()
+  @Prop({ type: String, required: true })
+  name: string;
+
+  @ApiProperty({ example: '0x123..', required: true })
+  @IsString()
+  @Prop({ type: String, required: true })
+  creator: string;
```

Comment:
I thought it might be better for our analytics ( to know who made how many and which communities), and also if you noticed on Telegram when you create a channel the creator is different from other admins and have more access from them, so it might be useful for us in the future, but we can remove it if want to.

Reply:
Hmm. Yes, I see what you mean, good idea. Then we would need to add "login with eth" to the community setup form to get the creator address. But maybe that is good anyway. Then @nebs-dev can add the creator address to the list of owners as well.

SetupWeb then works like this:
1. Login with ETH to be able to fill out creation form
2. Logged in address is added as creator and to list of owners
3. User is given the opportunity to add more addresses to the owners list

I will adjust wireframes

Reply:
Yeah exaclty